src/components/BaseTextField/index.tsx

```diff
@@ -71,11 +71,34 @@ export interface BaseTextFieldProps extends BaseComponentProps {
   onMouseLeave?: ((event: React.MouseEvent<HTMLInputElement | HTMLTextAreaElement>) => void);
 }
 
+/**
+ * This is a hack to allow us to accept jsx elements in the description prop.
```

Comment:
do we plan on updating the Fabric component to allow onRenderDescription without a description?

Reply:
I'd like to, but would prefer not to block this change on it.

Reply:
sounds good, can you add an issue to track it?

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/issues/4669